apps/sms/views/conversation/js/conversation.js

```diff
@@ -622,29 +625,31 @@ var ConversationView = {
     // to slide correctly. Bug 1009541
     this.cancelEdit();
 
-    if (Navigation.isCurrentPanel('thread')) {
```

Comment:
This was a quite futile optimization (not executing this if we're in the composer), so I removed it instead. "nodes" will likely be empty if we're in the composer.